test/functional/test_ivn.py

```diff
@@ -93,21 +104,42 @@ def _start_node(self, node_info):
         fake_node_obj.start()
         return fake_node_obj
 
-    def node_config(self, node_name, ns_name, boot_image):
+    def start_node_with_config(self, node_name, ns_name, boot_image, iso):
         fake_node = None
         fake_node = copy.deepcopy(FakeConfig().get_node_info())
         fake_node['name'] = node_name
         fake_node['namespace'] = ns_name
         fake_node['compute']['storage_backend'][0]["drives"][0]["file"] = boot_image
-        fake_node["compute"]["networks"][0]["port_forward"] = [{"outside": 8022, "inside": 22, "protocal": "tcp"}]
+        if "test0" in node_name:
+            fake_node['compute']['cdrom'] = {}
+            fake_node['compute']['cdrom']['file'] = iso
+            fake_node["compute"]["networks"].append({"device": "e1000", "network_mode": "bridge", "network_name": "br0",
+                                                     "mac": "00:60:16:93:b9:1d"})
+            fake_node["compute"]["networks"][0]["port_forward"] = [{"outside": 8022, "inside": 22, "protocal": "tcp"}]
+            fake_node["compute"]["networks"][0]["mac"] = "00:60:16:93:b9:2a"
+        if "test1" in node_name:
+            fake_node['compute']['cdrom'] = {}
+            fake_node['compute']['cdrom']['file'] = iso
+            fake_node["compute"]["networks"].append({"device": "e1000", "network_mode": "bridge", "network_name": "br0",
+                                                     "mac": "00:60:16:93:b9:1a"})
+            fake_node["compute"]["networks"][0]["port_forward"] = [{"outside": 8022, "inside": 22, "protocal": "tcp"}]
+            fake_node["compute"]["networks"][0]["mac"] = "00:60:16:93:b9:2d"
+        print "fake_node"
+        print fake_node
```

Comment:
it's better to remove "print".  print should be debug purpose only.

Reply:
okay!